src/components/Utils/normalizeItem.js

```diff
@@ -0,0 +1,7 @@
+// eslint-disable-next-line no-extend-native
+String.prototype.normalizeItem = function () {
```

Comment:
This is a big no-no as it could be really dangerous to modify an object's prototype.

Let's create this as a regular function instead. It could live in a `/helpers` file or something like that.

Reply:
@alejandronanez Thank you very much for the suggestion! I didn't know that you had to be careful with the prototype, it's still not very clear to me why. I continue to learn every day and it's great. The corresponding changes have already been made.

Reply:
This resource has a pretty good answer: https://flaviocopes.com/javascript-why-not-modify-object-prototype/ 
Also, please remove this file :)

Reply:
@indiragp, Thank you very much for the resource! The file has already been removed, it has been renamed as helpers.js taking into account the suggestion of @alejandronanez, if you want you can see the latest changes.

Reply:
Sorry y'all for leaving you hanging a couple of days ago 😅, I was on a hurry and from my 📱.

The article @indiragp shared is great, bookmark it for future reference :)

This operation is a big no-no because once you modify an Object's Prototype you _hide_ information and could easily override an existing method, for instance:

```js
// Array prototype has a `map` method

[1,2,3].map(n => n +1) // [2,3,4]

// But, if you decide to implement your own map like this
Array.prototype.map = () => 'hello';
// and then you do
[1,2,3].map(n => n + 1)
// You'll get "hello" back!
```

Happy to talk more about this if you still have questions! 😄 @Lukaina @macosta08

Reply:
Thanks a lot @alejandronanez @indiragp 👍